src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.tsx

```diff
@@ -110,10 +128,18 @@ const BlastInputSequence = (props: Props) => {
     }
   };
 
+  const isInputValid = input
+    ? checkSequenceValidity(input, sequenceType)
```

Comment:
Wouldn't it be better to validate the sequence `onBlur` instead of every keyups?
I wonder if it would annoy the users if is starts to complain as soon as the use inputs the first character.

Reply:
It absolutely would, but this is something that I was arguing about with Andrea during the planning meeting, and she insisted that we do it while typing. I was hoping somebody other than me would argue this point with Andrea as well, but no-one joined 🤷‍♂️